src/encoded/static/components/search.js

```diff
@@ -1047,7 +1047,12 @@ export const FacetList = (props) => {
     }
 
     // Combine facets from search results with special facets, and treat them mostly the same.
-    const allFacets = SpecialFacetRegistry.Facet.getFacets().concat(facets);
+    let allFacets = SpecialFacetRegistry.Facet.getFacets().concat(facets);
+
+    // Biochemical inputs facet exists for annotations but we only want to display it for the Encyclopedia page
+    if (context['@type'].indexOf('Encyclopedia') === -1) {
+        allFacets = allFacets.filter((facet) => facet.field !== 'biochemical_inputs');
+    }
```

Comment:
Since we got rid of all the special cases in the generic facet code, I’d like to avoid putting special cases back. You can have `BiochemicalInputsFacet` supress its own display if its `results` property doesn’t have “Encyclopedia” in its `@type[0]`. Then you don’t need this special-case code.

Reply:
I also fixed this.